webapp/src/main/resources/public/js/ict/Ict.js

```diff
@@ -87,19 +92,44 @@ class Ict {
             node = node.parentNode;
         }
 
-        if (!node.classList.contains("mojito-ict-string")) {
-           try {
-                node.className += " mojito-ict-string";
-            } catch (e) {
+        var mtRequired = textUnits[0]['isTranslated'] === 'false';
+        if (this.mtEnabled && mtRequired){
+            if (!node.classList.contains("mojito-ict-string")) {
+                try {
+                    node.className += " mojito-ict-string";
+                } catch (e) {
+                }
+
+                node.addEventListener("mouseenter", (e) => {
+                    e.target.classList.add("mojito-ict-string-mt");
+                });
+
+                node.addEventListener("mouseleave", (e) => {
+                    e.target.classList.remove("mojito-ict-string-mt");
+                });
             }
 
-            node.addEventListener("mouseenter", (e) => {
-                e.target.classList.add("mojito-ict-string-active");
-            });
+            chrome.runtime.sendMessage(textUnits[0], function (response){
```

Comment:
indent `function (response){`

refactor suggestion, probably want to share the event registration code?

```
var mtRequired = this.mtEnabled && textUnits[0]['isTranslated'] === 'false';
eventListenerClassName = mtRequired  ? "mojito-ict-string-mt" : "mojito-ict-string-active"

if (!node.classList.contains("mojito-ict-string")) {
      try {
          node.className += " mojito-ict-string";
      } catch (e) {
      }

      node.addEventListener("mouseenter", (e) => {
          e.target.classList.add(eventListenerClassName);
      });

      node.addEventListener("mouseleave", (e) => {
          e.target.classList.remove(eventListenerClassName);
      });
  }

if (mtRequired) {
    chrome.runtime.sendMessage(textUnits[0], function (response){ 
...

```